adminapi/filters.py

```diff
@@ -1,5 +1,10 @@
-# This module must only include the filter classes.  Everything defined
-# in here are going to collected as the filters.
+from re import compile as re_compile, error as re_error
```

Comment:
Seems not necessary as `re_compile` is the same as `re.compile` or is it because of performance?